docs/root/intro/version_history.rst

```diff
@@ -94,6 +94,13 @@ Version history
   generate session cookies.
 * router: added START_TIME as one of supported variables in :ref:`header
   formatters <config_http_conn_man_headers_custom_request_headers>`.
+* router: added a :ref:`max_grpc_timeout <envoy_api_field_route.RouteAction.max_grpc_timeout>`
+  config option to specify the maximum allowable value for timeouts decoded from gRPC header field
+  `grpc-timeout`. If configured as 0, the maximum allowed timeout for gRPC requests is infinity.
```

Comment:
nit: I would just merge this nice explanation into the main field docs and just let people click on the link.

Reply:
Merged.